api/actions/createChangelist.js

```diff
@@ -5,26 +5,25 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+const Gus = require('../services/Gus');
+const { convertUrlToGusFormat } = require('./convertUrlToGusFormat');
 const GithubEvents = require('../modules/GithubEvents');
 const Issues = require('../services/Issues');
 
 module.exports = {
     eventName: GithubEvents.events.PULL_REQUEST_CLOSED,
     fn: async function(req) {
         const {
-            pull_request: { title, url, closed_at }
+            pull_request: { title, url }
         } = req.body;
         if (title.includes('@W-')) {
```

Comment:
I think we should use regex here to be more specific about a work item name e.g. `/@[Ww]-\d{5}/` <-- something like that

Reply:
Took care of this.